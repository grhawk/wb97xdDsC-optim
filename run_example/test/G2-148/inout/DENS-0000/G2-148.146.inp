 $BASIS NGAUSS=6 GBASIS=N31 $END
 $SYSTEM MWORDS=8 $END
 $SCF DIRSCF=.t. $END
 $CONTRL SCFTYP=ROHF RUNTYP=ENERGY DFTTYP=wB97X EXETYP=RUN MAXIT=200 
ICHARG=0 MULT=2 $END
 $DATA
 Should be setted!
 C1
 C     6.00     0.014223     0.543850     0.000000
 C     6.00     0.014223    -0.199742     1.291572
 C     6.00     0.014223    -0.199742    -1.291572
 H     1.00    -0.322890     1.575329     0.000000
 H     1.00     0.221417     0.459174     2.138477
 H     1.00     0.221417     0.459174    -2.138477
 H     1.00    -0.955157    -0.684629     1.484633
 H     1.00     0.767181    -0.995308     1.286239
 H     1.00     0.767181    -0.995308    -1.286239
 H     1.00    -0.955157    -0.684629    -1.484633
 $END
 $DFT NRAD=24 DDSC=.t. NLEB=110 $END