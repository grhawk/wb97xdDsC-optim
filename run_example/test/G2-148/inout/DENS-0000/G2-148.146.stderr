/var/spool/slurm/job10170/slurm_script: line 9: module: command not found
